### DISCUSSION TITLE
Try screen mirror on PC software

### DISCUSSION THREAD

**Discussion by Sudupatil** [Bodmer/TFT_eSPI#2785] [General]
Hi,

I would like to know is there any way we can see the tar containt on vb.net desktop application (or in feature web page).
 I am communicating with ESP32 with UDP protocol. I can give them command and esp dose things, if anyhow i can see that screen in application then that may be good.

If you have any better solution then please let me know

Sudarshan Patil

**Comment by Bodmer** [comment]
There is no built in cabability to do this. If the screen can be read then pixels could be read back and sent over a network (his is not something I have done).

**Comment by Bodmer** [comment]
This may be of interest:
https://github.com/Links2004/arduinoVNC

Note: I have not tried this so am not able to provide support.
